rskj-core/src/main/java/org/ethereum/config/Constants.java

```diff
@@ -89,6 +89,7 @@ public BlockDifficulty getFallbackMiningDifficulty() {
     }
 
     public BigInteger getDifficultyBoundDivisor() {
```

Comment:
Since this "constant" is no longer constant maybe we should find a way to remove it from here